packages/server/lib/browsers/firefox.ts

```diff
@@ -419,5 +421,38 @@ export async function open (browser: Browser, url, options: any = {}) {
     errors.throw('FIREFOX_COULD_NOT_CONNECT', err)
   })
 
+  // Override the .kill method for Windows so that the Browsers closes b/w specs
+  if (os.platform() === 'win32') {
+    browserInstance.kill = async (...args) => {
+      debug('closing firefox on Windows')
+
+      killFirefoxInstanceWin32(browserInstance)
+    }
+  }
+
   return browserInstance
 }
+
+const killFirefoxInstanceWin32 = (browserInstance) => {
+  let instanceKill = `TASKKILL /T /PID ${browserInstance.pid}`
+
+  let parentPidFile = browserInstance.spawnargs[browserInstance.spawnargs.length - 1].split('\\')
+  let parentPidWithFilename = parentPidFile[parentPidFile.length - 1].split('-')
+  let parentPid = parentPidWithFilename[parentPidWithFilename.length - 1]
```

Comment:
the pid here is actually the pid of the Cypress Electron process, which we don't want to kill